docs/guides/composer.mdx

```diff
@@ -0,0 +1,731 @@
+
+#### Add a conditional
+
+To add a conditional to a gate, drag the `if` operation to the gate and set the parameters in the Edit operation panel that automatically opens. You can also double-click a gate to access the Edit operation panel, and set the conditional parameters that way.
+
+#### Add a control modifier
+
+A control modifier yields a gate whose original operation is now contingent on the state of the control qubit. For more details, right-click the control modifier symbol in the operations catalog, then click Info.
+
+Drag the control modifier to a gate in order to add a control to it. A dot appears on the control qubit and a line connects it to the target qubit. To edit which qubit is the control or target, click the gate and select the Edit operation icon (or double-click the gate) to open the Edit operation panel, then specify your parameters. From the Edit operation panel, you can also remove a control from a qubit by clicking the x next to the qubit name.
+
+#### Visualize with phase disks throughout your circuit
+
+To visualize the state of all qubits at any point in your circuit, drag the phase disk icon from the operations catalog and place it anywhere in your circuit. A column of barrier operations and a column of phase disks are added (one barrier operation and phase disk per qubit). Hover over each phase disk to read the state of the qubit at that point in the circuit. Note that adding the phase disks does not alter your circuit; they are merely a visualization tool.
+
+Read more about the phase disk visualization [here.](#phase-disk)
+
+#### Export a circuit image
+
+To export an image of your circuit, select File → Export circuit image. The Export options window opens, where you can choose a theme (light, dark, white on black, or black on white), a format (.svg or .png), and whether you want to apply a line wrap. After you’ve chosen your options, click Export.
+
+### 3. Build your circuit with OpenQASM code
+
+<Admonition type="note">
+IBM Quantum Composer currently supports OpenQASM 2.0.
+</Admonition>
+
+* To open the code editor, click View → Panels → Code Editor.
+* See the [Composer operations glossary](#composer-operations-glossary) for OpenQASM references to gates and other operations.
+* You can define your own custom operations; see [Create a custom operation in OpenQASM](#create-a-custom-operation-in-openqasm).
+* For more information on using the OpenQASM language, including sample lines of code, see the original research paper, [Open Quantum Assembly Language](https://arxiv.org/abs/1707.03429). The [table](https://arxiv.org/pdf/1707.03429.pdf#page=9) of OpenQASM language statements from the paper is reproduced below. The OpenQASM grammar can be found in [Appendix A](https://arxiv.org/pdf/1707.03429.pdf#page=21) of the paper.
```

Comment:
We don't want to link to https://quantum.cloud.ibm.com/docs/en/guides/introduction-to-qasm?

Reply:
Ha, it didn't exist when this was written 🤣